pkgs/development/compilers/adoptopenjdk-bin/sources.json

```diff
@@ -146,27 +146,27 @@
       "jdk": {
         "hotspot": {
           "aarch64": {
-            "build": "8",
+            "build": "8.1",
```

Comment:
How can the build increase but the download stays the same?

Reply:
@SuperSandro2000 that's what AdoptOpenJDK API responds with:

```
$ curl -s https://api.adoptopenjdk.net/v2/latestAssets/releases/openjdk13 | jq 'map(select(.os == "linux" and .architecture == "aarch64" and .binary_type == "jdk"))[0]'
{
  "os": "linux",
  "architecture": "aarch64",
  "binary_type": "jdk",
  "openjdk_impl": "hotspot",
  "binary_name": "OpenJDK13U-jdk_aarch64_linux_hotspot_13.0.2_8.tar.gz",
  "binary_link": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.2%2B8/OpenJDK13U-jdk_aarch64_linux_hotspot_13.0.2_8.tar.gz",
  "binary_size": 206018212,
  "checksum_link": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/download/jdk-13.0.2%2B8/OpenJDK13U-jdk_aarch64_linux_hotspot_13.0.2_8.tar.gz.sha256.txt",
  "version": "13",
  "version_data": {
    "openjdk_version": "13.0.2+8",
    "semver": "13.0.2+8.1"
  },
  "heap_size": "normal",
  "download_count": 3076,
  "updated_at": "2020-01-21T09:21:39Z",
  "timestamp": "2020-01-21T09:21:39Z",
  "release_name": "jdk-13.0.2+8",
  "release_link": "https://github.com/AdoptOpenJDK/openjdk13-binaries/releases/tag/jdk-13.0.2%2B8"
}
```

see `semver` in `version_data`: that's what is being used [in `generate-sources.py`](https://github.com/NixOS/nixpkgs/blob/3ee1fb4f09/pkgs/development/compilers/adoptopenjdk-bin/generate-sources.py#L37) (I just run `generate-sources.py` to update `sources.json` for the reference, didn't edit it manually)

If I were to guess, maybe something changed in AdoptOpenJDK's build process for those versions, but the underlying source and expected output is the same?

Reply:
Maybe. I don't know. It just looked strange and maybe something is broken or not.

Reply:
Looks like https://github.com/NixOS/nixpkgs/pull/113120 landed into `master` now so updates to Java 8-15 releases are no longer part of this PR (sorry, I didn't notice there was already a PR for those updates), so this PR is purely about adding Java 16 releases now